mkdocs.yaml

```
@@ -54,22 +64,62 @@ plugins:
       handlers:
         # See: https://mkdocstrings.github.io/python/usage/
         python:
-          paths: [../src]
+          paths: [src]
+          import:
+            - url: https://docs.python.org/3/objects.inv
+              domains: [py, std]
+          options:
+            docstring_options:
+              ignore_init_summary: true
+            # docstring_section_style: list
+            filters: ["!^_"]
+            heading_level: 1
+            inherited_members: true
+            merge_init_into_class: true
+            parameter_headings: true
+            separate_signature: true
+            show_root_heading: true
+            show_root_full_path: false
+            show_signature_annotations: true
+            # show_source: false
+            show_symbol_type_heading: true
+            show_symbol_type_toc: true
+            signature_crossrefs: true
+            # summary: true
 
 markdown_extensions:
-  - admonition
-
+  - admonition # https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks
   # - tables
+  - pymdownx.extra
+  - pymdownx.highlight:
+      pygments_lang_class: true
+      linenums: true
+  - pymdownx.superfences
   - pymdownx.snippets:
       check_paths: true
+  - pymdownx.emoji:
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
   - pymdownx.arithmatex:
       generic: true
 
 
+
 extra_javascript:
   - javascripts/mathjax.js
   - https://unpkg.com/mathjax@3/es5/tex-mml-chtml.js
 
-# extra:
+extra:
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/phlowers
```

I suggest adding a `name` attribute to each of these links, in order to accessibility (for instance for people using screenreaders). For example:

```suggestion
    - icon: fontawesome/brands/github
      link: https://github.com/phlowers
      name: phlowers on Github
```

From [mkdocs documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#+social.name):
> name:
> domain name from link, if available This property is used as the link's title attribute and can be set to a discernable 
> name to improve accessibility